ipfx/attach_metadata/sink/nwb2_sink.py

```diff
@@ -236,36 +235,3 @@ def serialize(self, targets: Optional[OneOrMany[Dict[str, Any]]] = None):
                 file_.write(self._data.getvalue())
 
         self._reload_nwbfile()
-
-
-def set_container_sources(
```

Comment:
I'm going to invoke [Chesterton's Fence](https://fs.blog/chestertons-fence/) here and ask for an explanation of what this existing code does? Why is it safe to remove this now but not previously?

Reply:
I agree with the invocation.
But I've been looking through #361 and the commit making the changes (1aeb641) and I am yet to find a good source of documentation for why the method existed.
There are a couple of internal confluence pages that I'm trying to get access to in case they are enlightening.
If you have suggestions for where to look or who to check with, please let me know.